<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(7, 3, 30);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color: #fff; font-size:24px;font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Sales Analysis</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">c</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: #fff; font-size:14px;font-weight:bold;
margin: 3px;</string>
         </property>
         <property name="text">
          <string>Tables</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sales_table_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:#fff; border-radius:10px;
border: 1px solid gray;
border-radius:2px;
font-size:14px;
padding: 2px 2px 2px  2px;
position:relative;
margin: 2px 6px;
}

QPushButton:hover{
	background-color:rgb(0, 0, 127)
}
</string>
         </property>
         <property name="text">
          <string>Sales table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sale_numeric_table_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:#fff; border-radius:10px;
border: 1px solid gray;
border-radius:2px;
font-size:14px;
padding: 2px 2px 2px  2px;
position:relative;
margin: 2px 6px;
}

QPushButton:hover{
	background-color:rgb(0, 0, 127)
}
</string>
         </property>
         <property name="text">
          <string>Sale Numeric Table</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>235</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color: #fff; font-size:14px;font-weight:bold;
margin: 3px;</string>
         </property>
         <property name="text">
          <string>Correlations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="product_line_city_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:#fff; border-radius:10px;
border: 1px solid gray;
border-radius:2px;
font-size:14px;
padding: 2px 2px 2px  2px;
position:relative;
margin: 2px 6px;
}

QPushButton:hover{
	background-color:rgb(0, 0, 127)
}
</string>
         </property>
         <property name="text">
          <string>Product Line - City</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="product_line_gender_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:#fff; border-radius:10px;
border: 1px solid gray;
border-radius:2px;
font-size:14px;
padding: 2px 2px 2px  2px;
position:relative;
margin: 2px 6px;
}

QPushButton:hover{
	background-color:rgb(0, 0, 127)
}
</string>
         </property>
         <property name="text">
          <string>Product Line - Gender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="city_gender_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:#fff; border-radius:10px;
border: 1px solid gray;
border-radius:2px;
font-size:14px;
padding: 2px 2px 2px  2px;
position:relative;
margin: 2px 6px;
}

QPushButton:hover{
	background-color:rgb(0, 0, 127)
}
</string>
         </property>
         <property name="text">
          <string>City - Gender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="city_client_type_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:#fff; border-radius:10px;
border: 1px solid gray;
border-radius:2px;
font-size:14px;
padding: 2px 2px 2px  2px;
position:relative;
margin: 2px 6px;
}

QPushButton:hover{
	background-color:rgb(0, 0, 127)
}
</string>
         </property>
         <property name="text">
          <string>City - Client Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="product_line_client_type_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:#fff; border-radius:10px;
border: 1px solid gray;
border-radius:2px;
font-size:14px;
padding: 2px 2px 2px  2px;
position:relative;
margin: 2px 6px;
}

QPushButton:hover{
	background-color:rgb(0, 0, 127)
}
</string>
         </property>
         <property name="text">
          <string>Product  Line - Client Type</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>234</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color:#fff;</string>
      </property>
      <property name="currentIndex">
       <number>7</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color: rgb(7, 3, 30); font-size:24px;font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Sales Table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="sales_table"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color: rgb(7, 3, 30); font-size:24px;font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Sales Numeric Table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="sales_numeric_table"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">color: rgb(7, 3, 30); font-size:24px;font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Correlation Between City and Product Line</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QTableWidget" name="city_product_line_table"/>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget_2" native="true">
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255, 255, 255)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="styleSheet">
             <string notr="true">color: rgb(7, 3, 30); font-size:24px;font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Correlation Between Gender and Product Line</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QTableWidget" name="gender_product_line_table"/>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget_3" native="true">
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255, 255, 255)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">color: rgb(7, 3, 30); font-size:24px;font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Correlation Between City and Gender</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QTableWidget" name="city_gender_table"/>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget_4" native="true">
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255, 255, 255)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="styleSheet">
             <string notr="true">color: rgb(7, 3, 30); font-size:24px;font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Correlation Between City and Client Type</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QTableWidget" name="city_client_type_table"/>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget_5" native="true">
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255, 255, 255)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_7">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="styleSheet">
             <string notr="true">color: rgb(7, 3, 30); font-size:24px;font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Correlation Between Product Line and Client Type</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_5">
            <item row="0" column="0">
             <widget class="QTableWidget" name="product_line_client_type_table"/>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget_6" native="true">
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255, 255, 255)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_8">
       <widget class="QWidget" name="widget_7" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>1190</width>
          <height>746</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../../Downloads/analysis.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
